src/ExpenseTrackerGrupo2/Controllers/IncomeController.cs

```diff
@@ -0,0 +1,143 @@
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using ExpenseTrackerGrupo2.Business.Services.Interfaces;
+using ExpenseTrackerGrupo2.Business.Services.Mappers.Requests.Incomes;
+
+namespace ExpenseTrackerGrupo2.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
```

Comment:
I suggest to change our path to api/v1/income/[controller]

Reply:
Done.